frontend/www/js/omegaup/components/problem/Collection.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <h1 class="card-title">{{ T.collectionTitle }}</h1>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import T from '../../lang';
+import { types } from '../../api_types';
+import * as ui from '../../ui';
+
+export default class problemCollection extends Vue {
```

Comment:
nit: las clases deben tener el mismo nombre que el archivo. y falta el `@Component`.

```suggestion
@Component
export default class Collection extends Vue {
```